### DISCUSSION TITLE
why don't I see the search icon

### DISCUSSION THREAD

**Discussion by Ralff212** [pimcore/pimcore#16276] [General]
![2023-11-21_102805](https://github.com/pimcore/pimcore/assets/104589439/fe65880c-e7dd-45f8-8bf5-e0bc83811c56)
Hello. i have a Pimcore11, i am trying to understand what i am doing wrong when building a category search object. 
I have such an object , I assigned it to the objects "category" unfortunately I do not have a magnifying glass icon to search the created objects. 
I can only drag a specific category to the field and then it works but no search. What am I doing wrong?

**Comment by doncatnip** [comment]
I have the same problem ever since updating to Pimcore 11. There is no search icon anywhere, not for relation fields nor in the global toolbar. Have you been able to solve the problem?

**Comment by fashxp** [comment]
Do you guys have activated / installed `PimcoreSimpleBackendSearchBundle`? 
See also [Upgrade notes](https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/Updating_Pimcore/V10_to_V11#activate-extracted-bundles-in-configbundlesphp-if-necessary)

**Comment by doncatnip** [comment]
I added the bundle to bundle.php. It did not work because it claimed it cannot find the Installer. What does not seem to be mentioned is that the cache needs to be cleared with bin/console cache:clear after bundle.php has changed.

**Reply by fashxp** [reply]
that is symfony behavior ... the container needs to be rebuilt after changing configs or the bundle.php ... and when in prod environmentf, it is required to call the cache clear command. 

did it work after rebuilding the container?

**Reply by doncatnip** [reply]
Yes, thx.
